.github/workflows/ax.yml

```diff
@@ -63,8 +63,22 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
+      - name: timestamp
+        id: timestamp
+        shell: bash
+        run: echo "::set-output name=timestamp::`date -u +'%Y-%m-%dT%H:%M:%SZ'`"
+      - name: ccache
+        # don't use ccache for debug builds
+        if: matrix.config.build == 'Release'
+        id: ccache
+        uses: actions/cache@v2
+        with:
+          path: /tmp/ccache
+          key: linux-ax${{ matrix.config.image }}-${{ matrix.config.cxx }}-${{ steps.timestamp.outputs.timestamp }}
+          restore-keys: linux-ax${{ matrix.config.image }}-${{ matrix.config.cxx }}-
       - name: build
         run: >
+          CCACHE_DIR=/tmp/ccache
```

Comment:
Put this in the `env:` of each job? You can then remove it from everywhere

Reply:
Done.